src/patent_fetcher.py

```diff
@@ -28,9 +28,14 @@ def conduct_search(title: str, author=False, status=False, expiry=False, lim=100
 
 
 @celery.task(name="download_patents")
-def download_patents(patent_list: List[USApplication], destination: str):
+def download_patents(destination: str,
```

Comment:
Don't like this function signature yet but a list of objects does not fly with Celery, see https://github.com/IshanMi/reverse-image-search-patent/issues/12